db/models/user.js

```diff
@@ -1,6 +1,6 @@
 import mongoose from "mongoose";
 import bcrypt from "bcrypt";
-import jwt from "jsonwebtoken";
+ 
 
 const schema = new mongoose.Schema(
```

Comment:
Check PR #6 for the schema structure, take a pull, and update this

Reply:
https://github.com/DuolearnHQ/Tasky/pull/11#discussion_r1440393675